plugins/chat/spec/services/chat/auto_join_channels_spec.rb

```diff
@@ -0,0 +1,359 @@
+# frozen_string_literal: true
+
+RSpec.describe Chat::AutoJoinChannels do
+  describe ".call" do
+    subject(:job) { described_class.call }
```

Comment:
This is a nitpick, but usually, we use `result` here (as the result of the service call is a result object). This could also be `service`, but here this is not a job :slightly_smiling_face: